frb/builds/build_frbs.py

```diff
@@ -123,6 +150,60 @@ def frb_190523():
     path = resource_filename('frb', 'data/FRBs')
     frb190523.write_to_json(path=path)
 
+def frb_190608():
+    """Bhandari+20, ApJL --
+    """
+    fname = 'FRB190608'
+    frb190608 = frb.FRB(fname, "J221604.74-075353.6",  # Pulled from Slack on 2019 Sep 20
```

Comment:
Would it be preferable to use the updated position?

Reply:
Do you mean from Cherie?  
If so, that's not public yet.

Or is there a new on in Shivani's paper?

Reply:
No. It looks like it's unchanged.